command/persign.go

```diff
@@ -0,0 +1,134 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/peak/s5cmd/v2/log/stat"
+	"github.com/peak/s5cmd/v2/storage"
+	"github.com/peak/s5cmd/v2/storage/url"
+)
+
+var persignHelpTemplate = `Name:
+	{{.HelpName}} - {{.Usage}}
+
+Usage:
+	{{.HelpName}} [options] source
+
+Options:
+	{{range .VisibleFlags}}{{.}}
+	{{end}}
+Examples:
+	1. Print a remote object url to stdout
+		 > s5cmd {{.HelpName}} s3://bucket/prefix/object
+
+	2. Print a remote object url with a specific expiration time to stdout
+		 > s5cmd {{.HelpName}} --expire 24h s3://bucket/prefix/object
+`
+
+func NewPersignCommand() *cli.Command {
+	cmd := &cli.Command{
+		Name:     "persign",
```

Comment:
Thank you for the contribution. Did you mean [presign](https://docs.aws.amazon.com/cli/latest/reference/s3/presign.html)?

Reply:
Yes, I have resubmitted.